components/profile-controller/controllers/profile_controller.go

```diff
@@ -325,109 +329,91 @@ func (r *ProfileReconciler) appendErrorConditionAndReturn(ctx context.Context, i
 func (r *ProfileReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&profilev1.Profile{}).
-		Owns(&istiorbac.ServiceRole{}).
-		Owns(&istiorbac.ServiceRoleBinding{}).
 		Owns(&corev1.Namespace{}).
 		Owns(&corev1.ServiceAccount{}).
 		Owns(&rbacv1.RoleBinding{}).
 		Complete(r)
 }
 
-// updateIstioRbac create or update Istio rbac resources in target namespace owned by "profileIns". The goal is to allow service access for profile owner
-func (r *ProfileReconciler) updateIstioRbac(profileIns *profilev1.Profile) error {
+func (r *ProfileReconciler) getSecurityCondition(profileIns *profilev1.Profile) []*istioSecurity.Condition {
+	namespaceCondition := istioSecurity.Condition{
+		Key:    "source.namespace",
+		Values: []string{profileIns.Name},
+	}
 	logger := r.Log.WithValues("profile", profileIns.Name)
```

Comment:
go vet ./...
# github.com/kubeflow/kubeflow/components/profile-controller/controllers
controllers/profile_controller.go:343:2: logger declared but not used